content/blog/projects/finished/kraksat.md

```diff
@@ -0,0 +1,36 @@
+---
+lastmod: 2023-11-8T4:21:00Z
+head:
+  meta:
+    - name: keywords
+      content: kraksat, cosmos, satellite, iss
+    - name: robots
+      content: noindex nofollow
+    - name: author
+      content: Andrzej Piotrowski
+    - name: copyright
+      content: © 2023 KN Integra
+    - name: createdAt
+      content: 2023-11-8T4:21:00Z
+---
+
+
+<p>
+        <img src="https://github.com/Grallistrix/integra.content_RzecznikPatches/assets/116823604/9966c763-d651-483e-b9c4-ba40aa66b6c2" alt="Logo KRAKSAT" style="max-height: 350px;  display: block; margin: 0; padding-left: 120px">

```

Comment:
Use correct Markdown syntax.
If exact sizes are required, request a new component in the `integra.front` repo